ios/linkedin-sdk.framework/Headers/LISDKErrorCode.h

```diff
@@ -0,0 +1,52 @@
+//
+//  LISDKErrorCode.h
+//
+//  Copyright (c) 2015 linkedin. All rights reserved.
```

Comment:
I'm surprised this isn't commented using /** **/. It's also poorly styled: change to LinkedIn.